subsys/bluetooth/Kconfig

```diff
@@ -83,6 +83,34 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_EXT_ADV
+	bool "Extended Advertising and Scanning support"
+	help
+	  Select this to enable Extended Advertising API support.
+	  This enables support for advertising with multiple advertising sets,
+	  extended advertising data, and advertising in long range mode.
+	  It enables support for receiving and scanning extended advertising
+	  data, and packets sent in long range mode.
+	  It enables establishing connections in long range mode.
+
+if BT_EXT_ADV
+config BT_EXT_ADV_LEGACY_SUPPORT
+	bool "Support starting advertising through legacy commands"
+	help
+	  Select this to enable the use of the Legacy Advertising HCI commands.
+	  This option should be used where the capabilities of the controller
+	  is not known.
+
+config BT_EXT_ADV_MAX_ADV_SET
+	int "Maximum number of simultaneous advertising sets"
+	range 1 1
```

Comment:
In the places I've looked, the code does seem to deal with the `adv_pool` containing more than one element. Where exactly is the support for more than one instance still missing?

Reply:
There are a few `/* TODO: handle multiple advertising set */` and `/* TODO: Foreach adv set */`
 around. The largest obstacle so far is the one in `enh_conn_complete, with multiple advertising set I know need to get the correct one. We might have to move some of the connected handling to the `le_adv_set_terminated` event instead, because only there we have both `adv_handle` and `conn_handle`.

Reply:
Also need to move the BT_DEV_ADVERTISING flag and others from `bt_dev.flags` to `adv->flags`,